### DISCUSSION TITLE
How to install scrot on Amazon Linux (AWS Workspaces)?

### DISCUSSION THREAD

**Discussion by Muhammad-Yousef** [resurrecting-open-source-projects/scrot#186] [Q&A] [answered]
Hi there,

I am using Amazon workspaces and I wanna launch an app on it that has scrot as a dependency.
The thing is, I can't figure out what is the right command to install it on Amazon Linux Distro.
Anybody can help me please?

TIA

**Comment by N-R-K** [comment, marked as answer]
>  The thing is, I can't figure out what is the right command to install it on Amazon Linux Distro.

You'll need to look up your distro's manual and documentation. If scrot isn't packaged on your distro, then you can compile from source [by following the build instruction](https://github.com/resurrecting-open-source-projects/scrot/#building).

If you run into any issue when building scrot, we can try to help. But distro specific things like "how to install X" is out of scope for this project, that's a question better directed at your distro's community forum (or similar).
